### PR TITLE
chore: update babel config and build

### PR DESCRIPTION
**Pull request by mobot11**
### Description
Library now compiles to the `lib` directory. Also compiles to commonjs. 

fixes https://github.com/SAP/fundamental-react/issues/317

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,14 +29,28 @@ For an existing react application, follow the steps bellow:
...
+    ```scss
     @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
     ```
 
 1. Import components as needed. See [Component Documentation](https://sap.github.io/fundamental-react/) for examples and API details.
-    ```
-    import { Alert } from 'fundamental-react';
-    ```
+```javascript
+import { Alert } from 'fundamental-react/lib/Alert/Alert';
```
Comment (jeffredodd) [inline README.md]: Shouldn't `import Alert from 'fundamental-react/lib/Alert/Alert'` also work in this case?

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    env: {
+        production: {
+            presets: [
+                [
+                    '@babel/preset-env',
+                    {
+                        targets: {
+                            ie: '11'
+                        }
+                    }
```
Comment (jeffredodd) [inline babel.config.js]: We should probably be using browserlist to set `@babel/preset-env`. https://github.com/SAP/fundamental-react/blob/master/package.json#L130 or maybe removal it all together if its not used anywhere but for babel. This was inherited from Create React App.

File: README.md (hunk 1 above)
Comment (greg-a-smith) [reply README.md]: @jeffredodd As of now anyway, there are no default exports in any of the source modules so named exports are all we have to work with.

Comment (mobot11) [reply README.md]: I don't think so since they aren't default exports.

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                ]
+            ],
+            plugins: productionPlugins
+        }
+    }
+};
```
Comment (greg-a-smith) [inline babel.config.js]: I have basically no idea what any of this is doing. 😏 I'd love a brief walkthrough of this config.

**Pull request merged**